event_sale_update_qty/wizards/event_sale_update_qty_wizard.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2025 Tecnativa - Pilar Vargas
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="view_event_sale_update_qty_wizard_form" model="ir.ui.view">
+        <field name="name">event.sale.update.qty.wizard.form</field>
+        <field name="model">event.sale.update.qty.wizard</field>
+        <field name="arch" type="xml">
+            <form string="Update Attendee Quantity">
+                <group>
+                    <field name="sale_order_line_id" invisible="1" />
+                    <field name="show_registrations" invisible="1" />
+                    <field name="event_id" />
+                    <field name="ticket_id" />
+                </group>
+                <group>
+                    <span colspan="2" class="o_form_label">Enter a new quantity</span>
+                    <div class="o_row">
+                        <div class="o_col">
+                            <field
+                                name="new_qty"
+                                widget="integer"
+                                style="width: 70px;"
+                            />
+                        </div>
+                        <div class="o_col" style="margin-left: 80px;">
+                            <a role="button" class="btn btn-secondary"><i
+                                    class="fa fa-check"
+                                    title="Validate quantity"
+                                /></a>
+                        </div>
```

Comment:
I find the “Validate quantity” icon somewhat confusing.

<img width="346" height="219" alt="ejemplo" src="https://github.com/user-attachments/assets/6b473b8c-c8f2-4ada-abe9-87d1d49a75f8" />

Perhaps it would be simpler to indicate the current quantity (read-only) and another field with the new quantity.